tests/PySys/local_mqtt_pub_sub/run.py

```diff
@@ -18,26 +18,37 @@
 class PySysTest(BaseTest):
     def execute(self):
         tedge = "/usr/bin/tedge"
+        sudo = "/usr/bin/sudo"
 
         sub = self.startProcess(
-            command=tedge,
-            arguments=["mqtt", "sub", "atopic"],
+            command=sudo,
+            arguments=[tedge, "mqtt", "sub", "atopic"],
             stdouterr="tedge_sub",
             background=True,
         )
 
+        # Wait for a small amount of time to give tedge sub time
+        # to initialize
         time.sleep(0.1)
```

Comment:
It's strange that this is required. `tedge mqtt sub` is doing nothing beyond a opening a socket and listening for message.

Reply:
That probably depends heavily on the environment and when the scheduler decides what to run. It failed only in 1% of the cases. 100ms is maybe a bit long but has no real impact on the test duration.